### DISCUSSION TITLE
Grouping information tags

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2685] [Ideas]
Available information for a given object is an ever increasing subject. It may be useful to group available information to make it easier to decide what the user really needs to be shown on screen, or what information are they supposed to provide. Groups as Identifiers, coordinates, physical properties, viewing properties (rise, set), etc. may pave the way to be able to show even more information tags in the future without really cluttering the screen. Just an idea.

**Comment by gzotti** [comment]
Yes, it has started to become a problem on small screens. We have short/default/customized info, currently it takes a minute to configure it and store the selection.

**Comment by diegorodriguezperegrin** [comment]
Yes, customizing which individual tags does the user want to see is a very nice feature. Maybe in the future, the possibility to add whole grups (as already seen with the seven Navigation tags) may make it even easier to decide which ones does the user need. Anyway, very easily configurable already. Many thanks.
